### PR TITLE
Upgrades OpenFst recipe to 1.7.5.

### PR DESCRIPTION
**Pull request by kylebgorman**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [na] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

I am the upstream maintainer of OpenFst. This upgrades the recipe to
version 1.7.5 and enables all extensions. Since this also causes the
binaries to be installed we move tests to basic shell tests instead of
the compilation-based test in the older version.

This is my first conda-forge submission so please be kind ;).

### CODE REVIEW

**Comment by kylebgorman** [comment]
Not sure what the CircleCI build 404 error is about---any tips?

**Comment by kylebgorman** [comment]
Gentle ping.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
-  dev_url: http://www.openfst.org/twiki/bin/view/Forum/FstForum
+    A library for constructing, combining, optimizing, and searching weighted 
+    finite-state transducers (FSTs)
+  doc_url: http://www.openfst.org/twiki/bin/view/FST/FstQuickTour
 
 extra:
   recipe-maintainers:
-    - mcernak
-    - 183amir
+    - kylebgorman
```
Comment (183amir) [inline recipe/meta.yaml:46]: Why are you removing former maintainers?

File: recipe/run_test.sh [hunk 2]
```diff
@@ -1,30 +0,0 @@
...
-  b.SetFinal(1, 0);
-
-  fst::Concat(&a, b);
-}
-EOF
-
-g++ -std=c++11 -lfst -I${PREFIX}/include -L${PREFIX}/lib main.cpp -o example
-export LD_LIBRARY_PATH=${PREFIX}/lib
-export DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib
-./example
```
Comment (183amir) [inline recipe/run_test.sh:30]: This was a good test. Why did you remove it?

**Comment by 183amir** [comment]
Hi @kylebgorman thank you for your contribution. I left some comments in the code. Also, why did you not re-render the project?

File: recipe/meta.yaml (hunk 1 above)
Comment (kylebgorman) [reply recipe/meta.yaml:46]: Accident. Restored.

File: recipe/run_test.sh (hunk 2 above)
Comment (kylebgorman) [reply recipe/run_test.sh:30]: I'm the upstream maintainer. This is an overly complex test; the software can easily be tested without compilation since it produces a host of useable binary endpoints.

**Review by kylebgorman — COMMENTED** [review]
I'd be glad to re-render it if you tell me what that means and how to do it ;)

File: recipe/LICENSE [hunk 3]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (183amir) [inline recipe/LICENSE]: Is this license shipped with openfst? If yes, we can use that one instead.

**Comment by 183amir** [comment]
For re-rendering, you can see the documentation here: https://conda-forge.org/docs/maintainer/infrastructure.html#admin-web-services

To do it, you just need to ask our bot, like this:
@conda-forge-admin, please rerender

File: recipe/LICENSE (hunk 3 above)
Comment (kylebgorman) [reply recipe/LICENSE]: Yes, same one. Does that mean I should just delete this file? And if so what if anything should I do to `meta.yaml`?

Comment (183amir) [reply recipe/LICENSE]: 1. yes, please delete it.
2. I don't think it will need a change.

**Review by 183amir — CHANGES_REQUESTED** [review]
Everything looks good except the license file

**Comment by kylebgorman** [comment]
Done!

**Comment by kylebgorman** [comment]
Linter wants the license file after all...should I revert the previous commit?

**Comment by 183amir** [comment]
Sorry for not being clear. I asked you not to change the meta.yaml file and only remove the license file. It should work like that. You can read about this option in https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file

**Comment by kylebgorman** [comment]
Ah, I get it now. It's the path _relative to the source directory_, so it's a reference to the license file in the tarball. Done.

**Review by 183amir — APPROVED** [review]

**Comment by kylebgorman** [comment]
Thanks for your review, Amir. I think what I've learned here will help me a lot in making more conda-forge contributions!

**Comment by 183amir** [comment]
Thank you @kylebgorman for your patience and your contribution. Please note that by merging this pull request, you will have write access to this repository. However, you should refrain from pushing to **master or any other branch** here. You should, as you did here, fork the project and open a pull request. Moreover, please do not open pull requests and merge them by yourself immediately. Always ask for feedback and if you don't get any in a few days, feel free to merge it yourself.
